src/EditorFeatures/Core/Extensibility/NavigationBar/WrappedNavigationBarItem.cs

```diff
@@ -35,16 +35,17 @@ internal WrappedNavigationBarItem(ITextVersion textVersion, RoslynNavigationBarI
 
         private static ImmutableArray<TextSpan> GetSpans(RoslynNavigationBarItem underlyingItem)
         {
-            return underlyingItem is RoslynNavigationBarItem.SymbolItem symbolItem && symbolItem.Location.InDocumentInfo != null
-                ? symbolItem.Location.InDocumentInfo.Value.spans
-                : ImmutableArray<TextSpan>.Empty;
-        }
-
-        private static TextSpan? GetNavigationSpan(RoslynNavigationBarItem underlyingItem)
```

Comment:
GetNavigationSpan effectively is just inlined to https://github.com/dotnet/roslyn/pull/57606/files#diff-519a5ce5b379e3685133221bb919d54aa28acd4d8deccfa2b495c5a2acfeaba8R74